### ISSUE TITLE
include FIR geometry information from ICAO

### ISSUE THREAD

**Issue by pedro2555** [pedro2555/vatsim-status-proxy#19]
https://v4p4sz5ijk.execute-api.us-east-1.amazonaws.com/anbdata/airspaces/zones/fir-list

```
params: {"api_key":"2a877ab0-4ed2-11e7-9b2e-d3182793b831", "firs": "", "format": "json"}
```

**Comment by pedro2555** [comment]
Implemented in 4e6f6ecbbe6a74b9c5aa79a2b73ef8eefe05ec3c

There are still some issues to work out:

- [ ] using regex is causing unwanted behaviour
`OAK` or Oakland Center, was actually matching `OAKX` or Kabul FIR; which is not correct

**Comment by pedro2555** [comment]
closed due to inactivty
